java/xtreemfs-servers/src/test/java/org/xtreemfs/test/mrc/OSDPolicyTest.java

```diff
@@ -339,14 +339,14 @@ public void testSortDCMapPolicy() throws Exception {
         
```

Comment:
Please also add a test for `FileNamePrefixPolicy` if that's doable in a sensible way.